toolchain/check/member_access.cpp

```diff
@@ -391,7 +391,6 @@ auto PerformMemberAccess(Context& context, Parse::NodeId node_id,
 
   return member_id;
 }
-
 auto PerformCompoundMemberAccess(Context& context, Parse::NodeId node_id,
```

Comment:
Shouldn't this be changed to support expressions that evaluate to a constant integer? This is where I would expect things like `t.(1)` to be supported.